src/main/java/blue/lapis/pore/impl/PoreServer.java

```diff
@@ -440,7 +464,13 @@ public void savePlayers() {
 
     @Override
     public boolean dispatchCommand(CommandSender sender, String commandLine) throws CommandException {
-        throw new NotImplementedException();
+        Preconditions.checkNotNull(sender, "sender");
+        Preconditions.checkNotNull(commandLine, "commandLine");
+
+        if (commandLine.startsWith("/")) {
```

Comment:
What if someone types '/mycomm' in the server console? I *think* the standard behavior is to not find the command, not remove the slash as done here